pkg/lib/bundle/exporter.go

```diff
@@ -0,0 +1,62 @@
+package bundle
+
+import (
+	"context"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+
+	"github.com/operator-framework/operator-registry/pkg/containertools"
+	"github.com/operator-framework/operator-registry/pkg/registry"
+	"github.com/otiai10/copy"
+	"github.com/sirupsen/logrus"
+)
+
+// BundleExporter exports the manifests of a bundle image into a directory
+type BundleExporter struct {
+	store         registry.Query
+	image         string
+	directory     string
+	containerTool string
+}
+
+func NewSQLExporterForBundle(store registry.Query, image, directory, containerTool string) *BundleExporter {
+	return &BundleExporter{
+		store:         store,
+		image:         image,
+		directory:     directory,
+		containerTool: containerTool,
+	}
+}
+
+func (i *BundleExporter) Export() error {
+
+	log := logrus.WithField("img", i.image)
+
+	folderName, err := i.store.GetBundleVersion(context.TODO(), i.image)
+	if err != nil {
+		return err
+	}
+	if folderName == "" {
+		// operator-registry does not care about the folder name
+		folderName = i.image
+	}
+	workingDir := filepath.Join(i.directory, folderName)
+	err = os.Mkdir(workingDir, 0777)
+	if err != nil {
+		return err
+	}
+	tmpDir, err := ioutil.TempDir("./", "bundle_tmp")
```

Comment:
Missing error handling.

Reply:
should we `defer os.RemoveAll()` the tmpDir like we did before?